docs/metrics.md

```diff
@@ -0,0 +1,120 @@
+openebs_pool_size # Size of pool
+openebs_pool_status # Status of pool (0, 1, 2, 3, 4, 5, 6)= {"Offline", "Online", "Degraded", "Faulted", "Removed", "Unavail", "NoPoolsAvailable"}
+openebs_read_latency # Read latency on replica
+openebs_rebuild_bytes # Rebuild bytes
+openebs_rebuild_count # Rebuild count
+openebs_rebuild_status # Status of rebuild on replica (0, 1, 2, 3, 4, 5, 6)= {"INIT", "DONE", "SNAP REBUILD INPROGRESS", "ACTIVE DATASET REBUILD INPROGRESS", "ERRORED", "FAILED", "UNKNOWN"}
+openebs_replica_status # Status of replica (0, 1, 2, 3) = {"Offline", "Healthy", "Degraded", "Rebuilding"}
+openebs_total_rebuild_done # Total number of rebuild done on replica
+openebs_sync_count # Total number of sync on replica
+openebs_sync_latency # Sync latency on replica
+openebs_total_failed_rebuild # Total number of failed rebuilds on replica
+openebs_total_read_bytes # Total read in bytes
+openebs_total_read_count # Total read io count
+openebs_total_rebuild_done # Total number of rebuild done on replica
+openebs_total_write_bytes # Total write in bytes
+openebs_total_write_count # Total write io count
+openebs_used_pool_capacity # Capacity used by pool
+openebs_used_pool_capacity_percent # Capacity used by pool in percent
+openebs_used_size Used # size of pool and volume
+openebs_volume_status # Status of volume (0, 1, 2, 3) = {"Offline", "Healthy", "Degraded", "Rebuilding"}
+openebs_write_latency # Write latency on replica
+openebs_zfs_command_error # zfs command error counter
+openebs_zfs_list_command_error # zfs list command error counter
+openebs_zfs_parse_error # zfs parse error counter
+openebs_zfs_list_failed_to_initialize_libuzfs_client_error_counter # Total no of failed to initialize libuzfs client error in zfs list command
+openebs_zfs_list_no_dataset_available_error_counter #  Total number of no datasets error in zfs list command
+openebs_zfs_list_parse_error # Total number of zfs list parse errors
+openebs_zfs_list_request_reject_count # Total number of rejected requests of zfs list
+openebs_zfs_stats_command_error # Total number of zfs command errors
+openebs_zfs_stats_parse_error_counter # Total number of zfs stats parse errors
+openebs_zfs_stats_reject_request_count # Total number of rejected requests of zfs stats
+openebs_zpool_list_command_error # Total number of zpool command error counter
+openebs_zpool_list_failed_to_initialize_libuzfs_client_error_counter # Total number of initialize libuzfs client error
+openebs_zpool_list_incomplete_stdout_error # Total number of incomplete stdout errors
+openebs_zpool_list_no_pool_available_error  # Total number of no pool available errors
+openebs_zpool_list_parse_error_count # Total number of parsing errors
+openebs_zpool_list_reject_request_count # Total number of rejected requests of zpool command
+```
+
+These metrics are collected are cStor podMonitor which declaratively specifies how groups of cStor pool pods should be monitored.
+
+# Monitoring Jiva Engine
```

Comment:
n_: move this into another doc. We can rename the files as metrics-jiva.md, metrics-cstor.md